Sources/NIOHTTP2/HTTP2ChannelHandler.swift

```diff
@@ -1109,6 +1129,7 @@ extension NIOHTTP2Handler {
         public var contentLengthValidation: ValidationState = .enabled
         public var maximumSequentialEmptyDataFrames: Int = 1
         public var maximumBufferedControlFrames: Int = 10000
+        public let maximumSequentialContinuationFrames: Int = 5
```

Comment:
This should be `var` as its a config that users can adjust before passing in

```suggestion
        public var maximumSequentialContinuationFrames: Int = 5
```